src/main/kotlin/frc/robot/lib/MarkerCommands.kt

```diff
@@ -0,0 +1,11 @@
+package frc.robot.lib
+
+import edu.wpi.first.wpilibj.DataLogManager
+import edu.wpi.first.wpilibj2.command.Command
+import edu.wpi.first.wpilibj2.command.Commands
+
+const val ABNORMAL_EVENT_NAME = "EVENT"
```

Comment:
I'd add the time since match start/match end. Did you check it's convenient in the console tab? I would also add error/warning so AdvantageScope will color it (check their docs for the exact wording).